camel-upgrade-recipes/src/main/resources/META-INF/rewrite/4.5.yaml

```diff
@@ -34,7 +34,7 @@ recipeList:
 ---
 type: specs.openrewrite.org/v1beta/recipe
 name: org.apache.camel.upgrade.camel45.UseExtendedCamelContextGetters
-displayName: Replace context.${method}(*) with context.getCamelContextExtension().${method}(*)
+displayName: Replace `context.${method}(*)` with `context.getCamelContextExtension().${method}(*)`
```

Comment:
This is the specific change I'd set out to apply; if nothing else of the changes here is accepted I hope you'll consider this.